Traceback (most recent call last):
  File "/shared/rc/defake/Deepfake-Slayer/scripts/DeepFake-Slayer/eval_forced_rgb.py", line 494, in <module>
    main()
  File "/shared/rc/defake/Deepfake-Slayer/scripts/DeepFake-Slayer/eval_forced_rgb.py", line 490, in main
    eval()
  File "/shared/rc/defake/Deepfake-Slayer/scripts/DeepFake-Slayer/eval_forced_rgb.py", line 446, in eval
    file.write('Found {0} total images with scores.\n'.format(TOTAL_RESULTS['lab'].shape[0]))
KeyError: 'lab'
